### PR TITLE
Prepare documentation rendering

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #131

### CODE REVIEW

File: Documentation/Documentation/Rendering/Index.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.. include:: ../../Includes.txt
+
+.. _documentation:
+
+==============
+Documentation rendering
+==============
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:7]: Shouldn't the underlining be the same length as the headline text?

File: Documentation/Documentation/Rendering/Index.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+.. _how-to-render-documentation-locally:
+
+How to render documentation locally?
+================
+
+In order to render documentation, first of all, clone repository:
+
+.. code-block:: none
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:16]: Can we use syntax highlighting for bash?

File: Documentation/Documentation/Rendering/Index.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+   git clone https://github.com/TYPO3-Documentation/tea.git
+
+then go to extension root:
+
+.. code-block:: none
+
+   cd tea
+
+
+and follow `this guide <https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/Quickstart.html>`_
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:27]: Maybe we can add a more descriptive title, e.g., "the TYPO3 documentation quickstart guide".

File: Documentation/Index.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+:Version:
+   |release|
+
+:Language:
+   en
+
+:Description:
+   This TYPO3 extension is an example of best practices in continuous integration and automated code checks, also
+   Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit.	writing unit and functional tests for Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit.
```
Comment (oliverklee) [inline Documentation/Index.rst:17]: There should only be one space after the comma.

Also, lines should be no longer than 80 characters.

File: Documentation/Index.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+   typo3-coding@oliverklee.de
+
+:License:
+   This extension is published under the
+   `GNU General Public License v2.0 <https://www.gnu.org/licenses/old-licenses/gpl-2.0.html>`__
+
+:Rendered:
+	|today|
+
+**TYPO3**
```
Comment (oliverklee) [inline Documentation/Index.rst:38]: Shouldn't these be headings of a lower level, i.e., underlined?

File: README.md [hunk 6]
```diff
@@ -88,6 +88,22 @@ which usually is the source for security checks.
 
 Instead, the projects (i.e., for TYPO3 installations) need to have the security checks.
 
+## Documentation rendering
+
+In order to render documentation, first of all, clone repository
```
Comment (oliverklee) [inline README.md:93]: Can we link the ReST documentation from the README? (IIRC, GitHub can render ReST.)

File: README.md [hunk 7]
```diff
@@ -88,6 +88,22 @@ which usually is the source for security checks.
...
+```
+git clone https://github.com/TYPO3-Documentation/tea.git
+```
+then go to extension root
+
+```
+cd tea
+```
+
+and follow [this guide](https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/Quickstart.html).
```
Comment (oliverklee) [inline README.md:104]: Ditto about the link title.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (lukaszuznanski) [reply README.md:93]: OK I will check this one. Maybe we can add it in separate PR with new issue, and merge this one first?

File: Documentation/Documentation/Rendering/Index.rst [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+.. _documentation:
+
+=======================
+Documentation rendering
+=======================
+
+.. _how-to-render-documentation-locally:
+
+How to render documentation locally?
+================
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:12]: Here, the length of the underlining should be the same as thaf ot the underlined text.

File: Documentation/Index.rst [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+:License:
+   This extension is published under the
+   `GNU General Public License v2.0 <https://www.gnu.org/licenses/old-licenses/gpl-2.0.html>`__
+
+:Rendered:
+	|today|
+
+=====
+TYPO3
+=====
```
Comment (oliverklee) [inline Documentation/Index.rst:41]: According to https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html#rest-sphinx-cheat-sheet, there may only be one heading per file (marked with overlining and underlining), and the others need to be of the level header 1 etc. (i.e., without overlining). Ditto for the other non-title headers in this file.

File: README.md [hunk 10]
```diff
@@ -88,6 +88,22 @@ which usually is the source for security checks.
 
 Instead, the projects (i.e., for TYPO3 installations) need to have the security checks.
 
+## Documentation rendering
+
+In order to render documentation, first of all, clone repository
+
+```
```
Comment (oliverklee) [inline README.md:95]: Please let's add the `bash` syntax highlighting to the new code blocks in this file.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Comment by lukaszuznanski** [comment]
@oliverklee please take a look :)

File: .gitignore [hunk 11]
```diff
@@ -8,3 +8,4 @@
 /Resources/Private/yarn.lock
 /Resources/Private/yarn-error.log
 /Resources/Private/package-lock.json
+/Documentation-GENERATED-temp/
```
Comment (oliverklee) [inline .gitignore:11]: We need a LF after the last line, and the lines should be sorted.

File: Documentation/Documentation/Rendering/Index.rst [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+=======================
+Documentation rendering
+=======================
+
+.. _how-to-render-documentation-locally:
+
+How to render documentation locally?
+====================================
+
+In order to render documentation, first of all, clone repository:
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:14]: clone -> clone the

File: Documentation/Documentation/Rendering/Index.rst [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+
+.. code-block:: bash
+
+   git clone https://github.com/TYPO3-Documentation/tea.git
+
+then go to extension root:
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:20]: to -> to the

File: Documentation/Documentation/Rendering/Index.rst [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+
+.. code-block:: bash
+
+   cd tea
+
+
```
Comment (oliverklee) [inline Documentation/Documentation/Rendering/Index.rst:26]: Only a single blank line, please.

File: README.md [hunk 15]
```diff
@@ -88,6 +88,22 @@ which usually is the source for security checks.
...
+then go to extension root
+
+```bash
+cd tea
+```
+
+and follow [the TYPO3 documentation quickstart guide](https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/Quickstart.html).
+
+
```
Comment (oliverklee) [inline README.md:106]: Only one blank line, please.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]

**Pull request merged**